pkg/crds/crds_test.go

```diff
@@ -0,0 +1,156 @@
+package crds
```

Comment:
@Javlopez thanks for adding the unit tests.

@travisn till now, for the tool we rely on CI to tests and have not written unit tests. Do you think in the future adding unit tests will be better or CI tests be good enough for validation? 

whatever we decided, we can definitely keep these  tests as a lot of effort have been gone in writing

Reply:
We really should have an integration test for every command so we know it works end-to-end. Unit tests are also generally faster to develop for smaller components or functions for more detailed variations, so we should have unit tests as well. The balance just depends on the command and the components being tested.